package.json

```diff
@@ -20,7 +20,7 @@
     "build": "npm run tsoa:build && tsc",
     "tsoa:build": "tsoa spec-and-routes",
     "tsoa:watch": "node --watch-path=./src ./node_modules/.bin/tsoa -- spec-and-routes",
-    "dev": "npm run tsoa:watch & NODE_ENV=dev node --import=tsimp/import --watch src/index.ts | pino-colada",
+    "dev": "npm run tsoa:watch & NODE_ENV=dev node --import=tsimp/import --watch src/index.ts",
```

Comment:
Mmm. mainly because it strips a lot of details so for logging JSON removed, ideally we could use something like https://github.com/pinojs/pino-pretty looks more configurable. Will put it back shortly just need to gather more logs for documentation

Reply:
Mmm. since will be going through all levels I think as a side task would be nice to have as part of this at least will give it a go

Reply:
https://digicatapult.atlassian.net/browse/VR-221 ended up creating a back log story, will need to expand on it